examples/solvers/chambolle_pock_tomography_tgv.py

```diff
@@ -47,12 +47,7 @@
 # --- Generate artificial data --- #
 
 # Create phantom
-def phantom_func(pts):
-    x, y = pts
-    sign = ((x < -10) | (x > 10) | (y < -10) | (y > 10)) * 3 - 2
-    return 0.5 * ((x / 20.0) * sign + 1)
-
-phantom = U.element(phantom_func)
+phantom = odl.phantom.tgv_phantom(U)
```

Comment:
Can we go with the usual `reco_space` here? Calling `A` something else like `fwd_op` would also be nice.